src/main/scala/io/viash/wrapper/BashWrapper.scala

```diff
@@ -117,6 +117,11 @@ object BashWrapper {
     mods: BashWrapperMods = BashWrapperMods(),
     debugPath: Option[String] = None
   ): String = {
+    // Escape substrings with pipe characters, meant to be used when using stripMargin.
+    // If the very first substring also starts with a pipe, strip the escaping off again as it will either become part of a previous string
+    // or would have its own | from the multiline string, and we don't want double pipes.
+    def escapePipes(s: String) = s.split("\n").map(_.replaceFirst("^(\\s*\\|)", "|$1")).mkString("\n").stripPrefix("|")
```

Comment:
This might not work when the first line starts with a pipe already because it will result in double pipes, correct?

Reply:
True! But that is why the `.stripPrefix("|")` is there

Reply:
When comparing:

```scala
x.split("\n").map(_.replaceFirst("^(\\s*\\|)", "|$1")).mkString("\n").stripPrefix("|")
```

    val res1: String =
    |foo
    | | one
    bar